roles/netbox/templates/docker-compose.yml.j2

```diff
@@ -54,24 +54,13 @@ services:
       - "traefik.enable=true"
```

Comment:
I'm not sure where netbox is being configured to use the expanded path that is being used in the healthcheck above, but likely that would also need to be amended?